### PR TITLE
chore: install ws-cleanup plugin

### PR DESCRIPTION
**Pull request by jazibhumayun**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A SRE team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a SRE ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/spaces/SRE/pages/28967861/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

**Comment by jazibhumayun** [comment]
Ran the plan plugin upgrade job log seems fine 
[https://build.testeng.edx.org/job/plan-plugin-upgrade-path/18/console](https://build.testeng.edx.org/job/plan-plugin-upgrade-path/18/console)

File: playbooks/roles/jenkins_data_engineering_new/defaults/main.yml [hunk 1]
```diff
@@ -267,6 +267,9 @@ de_jenkins_plugins_list:
   - name: 'workflow-step-api'
     version: '2.21'
     group: 'org.jenkins-ci.plugins.workflow'
+  - name: 'ws-cleanup'
+    version: '0.39'
```
Comment (HassanJaveed84) [inline playbooks/roles/jenkins_data_engineering_new/defaults/main.yml:271]: Can we go with the latest v0.40 ?

Comment (jazibhumayun) [reply playbooks/roles/jenkins_data_engineering_new/defaults/main.yml:271]: I think we can't until we upgrade jenkins version
I checked the jenkins version in pom.xml
latest v0.40 requires jenkins v2.235.5 https://github.com/jenkinsci/ws-cleanup-plugin/blob/ws-cleanup-0.40/pom.xml#L47

Our jenkins version is 2.222.3

Comment (jazibhumayun) [reply playbooks/roles/jenkins_data_engineering_new/defaults/main.yml:271]: v0.39 requires 2.164.3 or above https://github.com/jenkinsci/ws-cleanup-plugin/blob/ws-cleanup-0.39/pom.xml#L47
so we this should with our current version of jenkins

Comment (HassanJaveed84) [reply playbooks/roles/jenkins_data_engineering_new/defaults/main.yml:271]: Thanks for the explanation.

**Review by HassanJaveed84 — APPROVED** [review]

**Pull request merged**